### ISSUE TITLE
Failure (htpasswd module) with Debian Stretch

### ISSUE THREAD

**Issue by HanXHX** [HanXHX/ansible-nginx#28]
Due to this bug https://github.com/ansible/ansible/issues/20199 (fixed by https://github.com/ansible/ansible/pull/20202 but not merged), Debian Stretch cannot work with current Ansible version.

For now, Ansible version supported >= 2.2. But when this PR will be release, I need to:

* ~~change meta min version~~
* remove allow_failure in travis for stretch
* remove old ansible version on travis

**Comment by HanXHX** [comment]
OK in devel ansible branch.

TODO: wait ansible 2.3.2.0 ~~and change min version in meta/main.yml~~

**Comment by HanXHX** [comment]
TODO:
- remove failure on travis

```yaml
matrix:
  allow_failures:
    - env: PLATFORM='docker-debian-stretch'          ANSIBLE_VERSION='ansible>=2.3,<2.4'
  fast_finish: true

```

**Comment by HanXHX** [comment]
Fixed on Ansible 2.3.20
